### ISSUE TITLE
Support OpenChat 3.5?

### ISSUE THREAD

**Issue by kitckso** [vectorch-ai/ScaleLLM#29]
From discussion in issue #28, the chat template is embedded in code, so I think currently not supported?
Can support this model?

From official openchat/openchat_3.5:
GPT4 Correct User: Hello<|end_of_turn|>GPT4 Correct Assistant:

From TheBloke/openchat_3.5-AWQ:
GPT4 User: {prompt}<|end_of_turn|>GPT4 Assistant:

I’m not sure why it is different in the quantized model.

**Comment by guocuimi** [comment]
Yeah. It is not configurable right now. We will try to support it as soon as we can.
